### ISSUE TITLE
Update IC stock on hand mappings

### ISSUE THREAD

**Issue by wlthomson** [msupply-foundation/elmis-service#130]
Currently `stockInHand` is calculated using the following formula:

```
outgoingLine.beginningBalance - incomingLine.actualQuan + incomingLine.Cust_stock_received + incomingLine.Cust_lost_adjust;
```

Suggestion is to replace this with a direct mapping between `incomingLine.stock_on_hand`.

**Comment by josh-griffin** [comment]
👍

**Comment by wlthomson** [comment]
Fixed by #130.
